### ISSUE TITLE
Blue bubbles causing a facetime crash 

### ISSUE THREAD

**Issue by cameronaaron** [BlueBubblesApp/bluebubbles-server#635]
Blue bubbles causing a facetime crash that then kernel panics my machine and kills the server on the laptop running amphetamine and bluebubbles. Because its a laptop crashes mean I have to actually physically relogin 

[FaceTime-2024-03-16-175931.log](https://github.com/BlueBubblesApp/bluebubbles-server/files/14658094/FaceTime-2024-03-16-175931.log)
[FaceTime-2024-03-16-201210.log](https://github.com/BlueBubblesApp/bluebubbles-server/files/14658095/FaceTime-2024-03-16-201210.log)
[FaceTime-2024-03-17-114431.log](https://github.com/BlueBubblesApp/bluebub
[Email Log.log](https://github.com/BlueBubblesApp/bluebubbles-server/files/14658107/Email.Log.log)
bles-server/files/14658096/FaceTime-2024-03-17-114431.log)
[WindowServer-2024-03-17-020448.log](https://github.com/BlueBubblesApp/bluebubbles-server/files/14658097/WindowServer-2024-03-17-020448.log)
[WindowServer-2024-03-17-020528.log](https://github.com/BlueBubblesApp/bluebubbles-server/files/14658098/WindowServer-2024-03-17-020528.log)

**Comment by zlshames** [comment]
I have no idea what would cause this. So the crash doesn't happen so long as the FaceTime Private APi is disabled?

**Comment by cameronaaron** [comment]
yes

**Comment by zlshames** [comment]
> yes

Where did you get those logs from? They don't have much info in them that indicate an error. All it really shows is:

`"exception" : {"codes":"0x0000000000000000, 0x0000000000000000","rawCodes":[0,0],"type":"EXC_CRASH","signal":"SIGABRT"},`

The rest doesnt really point to anything. So what's happening is it's not just FaceTime that's crashing, its your entire user-profile?

**Comment by cameronaaron** [comment]
Yes it causes a full kernal panic which takes out the machine entirely these are the panic logs this only happens when the facetime private api is enabled

**Comment by cameronaaron** [comment]
Any ideas?

**Comment by zlshames** [comment]
> Any ideas?

nope, not a single one. This is the only report of this i've seen
